.github/workflows/build-and-test.yml

```diff
@@ -34,19 +33,23 @@ jobs:
       - name: Install Julia
         uses: julia-actions/setup-julia@v1
         with:
-          version: ${{ matrix.julia-version }}     
-        
-      - uses: actions/checkout@v2
+          version: ${{ matrix.julia-version }}  
+   
+      - uses: precice/setup-precice-action@testing # needs to be changed to main after merge
```

Comment:
So, actions are just sourced from any repository on GitHub? That's nice. We can then use it however we want and then it depends if and how much we advertise it to others.